src/lib/realtime.js

```diff
@@ -1,38 +1,53 @@
-import realtime from 'cozy-realtime'
+import CozyRealtime from 'cozy-realtime'
+import CozyClient from 'cozy-client'
```

Comment:
CozyClient is based on React, so we can't not import it at the beginning if we want to get React from the application, we have to require it on init, see https://github.com/cozy/cozy-bar/blob/master/src/index.jsx#L43

Reply:
All libraries using React/ReactDom can't be imported at the top of files like this,we have to do that if we don't want to use React from node_modules and give time to the bar to check if the application [gave it React through the dom](https://github.com/cozy/cozy-bar/blob/ecab3bd98ae578f0869f3468e29f9649b96aee64/config/aliases/globalReact.js)

Reply:
That should not be a problem, we talk about apps that do not use cozy-client yet anyways (otherwise they should have been able to init the bar with it)

Reply:
Since merely importing CozyClient and not any HOC, webpack should do its job and not import React, right ? I am going to test this to be sure, WDYT ?

Reply:
It should but I agree to test it to be sure. But if something would change on CozyClient side about React usage, we won't be protected here

Reply:
Do you have news on this ?